operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -61,7 +61,7 @@
    */
   @Description(
       "The directory containing the WebLogic domain configuration inside the container."
-          + " Defaults to /shared/domains/domains/<domainUID> if `domainHomeSourceType` is PersistentVolume."
+          + " Defaults to /shared/domains/<domainUID> if `domainHomeSourceType` is PersistentVolume."
           + " Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image."
```

Comment:
Same comment.

Reply:
This one is the source of truth and the others are generated as part of the build.